src/main/java/org/folio/services/piece/PieceClaimingService.java

```diff
@@ -1,11 +1,33 @@
 package org.folio.services.piece;
 
 import io.vertx.core.Future;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.dao.ExtensionRepository;
+import org.folio.dao.PieceClaimingRepository;
+import org.folio.dao.PostgresClientFactory;
+import org.folio.rest.persist.PostgresClient;
 
 public class PieceClaimingService {
+  private static final Logger log = LogManager.getLogger();
 
-  public Future<Integer> processClaimedPieces() {
-    return Future.succeededFuture(0);
+
+  private final PostgresClientFactory pgClientFactory;
+  private final ExtensionRepository extensionRepository;
+  private final PieceClaimingRepository pieceClaimingRepository;
+
+  public PieceClaimingService(PostgresClientFactory pgClientFactory, ExtensionRepository extensionRepository, PieceClaimingRepository pieceClaimingRepository) {
+    this.pgClientFactory = pgClientFactory;
+    this.extensionRepository = extensionRepository;
+    this.pieceClaimingRepository = pieceClaimingRepository;
+  }
+
+
+  public Future<Integer> processClaimedPieces(String tenantId) {
+    log.trace("processClaimedPieces, tenantId={}", tenantId);
```

Comment:
Basically this job will be executed only once a day, so it's okay to have info level here- in this log

Reply:
Increased up to info level